pkg/pool/bytes.go

```diff
@@ -0,0 +1,40 @@
+package pool
+
+// Bytes is a pool of byte slices that can be re-used.  Slices in
+// this pool will not be garbage collected when not in use.
+type Bytes struct {
+	pool chan []byte
+}
+
+// NewBytes returns a Bytes pool with capacity for max byte slices
+// to be pool.
+func NewBytes(max int) *Bytes {
+	return &Bytes{
+		pool: make(chan []byte, max),
+	}
+}
+
+// Get returns a byte slice size with at least sz capacity.
```

Comment:
This godoc comment should indicate the slice returned may contain garbage, not zeros.